### ISSUE TITLE
question

### ISSUE THREAD

**Issue by liuqinyong** [OrigamiSL/GBT#6]
你好，我根据参数跑了ETTh1 96步预测，实验结果一直上不去。是有哪里参数不对吗？
Namespace(ConvKer=3, INN=1, K=1, L=3, activation='gelu', attn='Full', auto_d_layers=1, base='legendre', batch_normalization=False, batch_size=32, c_out=7, checkpoints='./checkpoints/', cols=None, concat_len=0, criterion='Standard', d_layers=2, d_model=512, data='ETTh1', data_path='ETTh1.csv', dec_in=7, degree_of_polynomial=5, des='Exp', devices='0,1,2,3', dilation=1, distil=True, dropout=0.1, dropout_prob_theta=512, enc_in=7, factor=5, fd_model=32, feature_extractor='Attention', features='M', format='transformer', gpu=0, hidden_size=8, instance=False, inverse=False, itr=1, kernel=3, label_len=96, learning_rate=0.0001, loss='mse', lradj='type1', max_batch=512, mix=True, mode_select='random', model='GBT', modes=64, moving_avg=[24], n_blocks=[1, 1, 1], n_freq_downsample=[60, 8, 1], n_heads=8, n_hidden=512, n_layers=[2, 2, 2, 2, 2, 2, 2, 2, 2], n_pool_kernel_size=[4, 4, 4], n_s_hidden=0, n_x=0, num_groups=1, num_levels=3, num_of_harmonics=5, num_stacks=1, num_workers=0, output_attention=False, padding=0, patience=3, positionalEcoding=True, pred_len=96, reproducible=False, root_path='D:/研究生内容/数据来源', s_layers=[3, 2, 1], save_result=True, seasonality_blocks=5, seasonality_layer_size=512, seasonality_layers=5, seq_len=96, shared_weights=False, sigma=0.2, single_step_output_One=0, stack_num=3, target='OT', test_inverse=False, time=True, train=True, train_epochs=10, trend_blocks=5, trend_layer_size=512, trend_layers=5, use_RevIN=False, use_amp=False, use_gpu=True, use_multi_gpu=False, version='Fourier')
first stage|Mean|mse:0.6389647126197815, mae:0.5910306572914124|Std|mse:0.0, mae:0.0
second stage|Mean|mse:0.6295344829559326, mae:0.590165376663208|Std|mse:0.0, mae:0.0
